content/wiki/capitalization_scheme.md

```diff
@@ -0,0 +1,11 @@
++++
+title = "Capitalization scheme for Linebender projects"
++++
+
+All Linebender projects should follow this capitalization scheme:
+
+- The project's name should start with an uppercase letter when referring to the project itself.
+- The name should be all-lowercase where referring to the project as a Rust crate (for instance, in relation to crates.io) or as a Github repository.
+- In the latter two cases, it's encouraged to use backticks to put the project name in a codeblock, eg Vello vs `vello`.
```

Comment:
I think we could have an example like [vello#1](https://github.com/linebender/vello/issues/1), which we would not expect to be in a codeblock